x-pack/plugins/siem/server/lib/timeline/routes/import_timelines_route.ts

```diff
@@ -123,9 +122,9 @@ export const importTimelinesRoute = (
                       pinnedEventIds,
                       globalNotes,
                       eventNotes,
-                      templateTimelineId,
-                      templateTimelineVersion,
-                      timelineType,
+                      templateTimelineId = null,
```

Comment:
would be great if we could avoid default values here as well

Reply:
If I understand correctly, the templateTimelineId here is from user input, so we couldn't guarantee the property would always exist

Reply:
I see how you mean, put the fallback logic into classes now

Reply:
Great work with adding defaults in the classes 👍 So maybe we can remove those defaults from here?